### DISCUSSION TITLE
after run "net start DsExampleSvc", the port 8000 is not started

### DISCUSSION THREAD

**Discussion by chenjiajun1991** [ONLYOFFICE/DocumentServer#2005] [Q&A]
GET http://localhost/7.2.1-34/doc/__1test__2_.txt1669012504366/c/info?t=1669014445569 502 (Bad Gateway)

**Comment by ShockwaveNN** [comment]
Hi, please add more details in your message, we are now a wizard guessing your setup by one line
Whta's the os, how did you setup, is anything else on your system?

**Reply by chenjiajun1991** [reply]
Hi, the os is windows server 2012,  I followed this guide [https://helpcenter.onlyoffice.com/installation/docs-community-install-windows.aspx](url) to install it. I installed it on win10, and  run example the function is ok, but the same step was installed offline on window server 2012, and the port 8000 has not been started.
![image](https://user-images.githubusercontent.com/17824508/202995190-0475d116-3a33-4a47-9573-3cc940a37dfb.png)
![image](https://user-images.githubusercontent.com/17824508/202995466-00a80ede-b806-4cb0-a133-2e9af9e93551.png)
![image](https://user-images.githubusercontent.com/17824508/202995884-09a07d8e-c73d-4ce5-b79b-9312e5e0c0cd.png)
![image](https://user-images.githubusercontent.com/17824508/202996007-8f02c636-d054-428e-80f3-cf35930ea339.png)
the error " http://localhost/7.2.1-34/doc/__1test__2_.txt1669012504366/c/info?t=1669014445569 502 (Bad Gateway)"
I installed it offline, and when I encountered the default plugin, I chose to skip it, not sure if this has anything to do with it

**Reply by ShockwaveNN** [reply]
@igwyd Please take a look

**Comment by igwyd** [comment]
Hello @chenjiajun1991, i tried to reproduce your error, but my server started up and the test example works without errors on Windows server 2012, port 8000 is open. How version Postgre do you installed? Draw your attention we not supported Postgre 15 yet.
